src/components/common/program-section-templates/faq-program-section/ViewFaqProgramSection.module.scss

```
@@ -0,0 +1,62 @@
+    padding-top: 24px;
+    gap: 16px;
+
+    @media (min-width: 1440px) {
+        flex-direction: row;
+        justify-content: space-between;
+    }
+}
+
+.title {
+    @include type.h4-bold-uppercase;
+    text-transform: uppercase;
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)